packages/fx-core/resource/package.nls.json

```diff
@@ -529,6 +529,30 @@
   "core.copilot.addAPI.success": "%s have(has) been successfully added to %s",
   "core.copilot.addAPI.InjectAPIKeyActionFailed": "Inject API key action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
   "core.copilot.addAPI.InjectOAuthActionFailed": "Inject OAuth action to teamsapp.yaml file unsuccessful, make sure the file contains teamsApp/create action in provision section.",
+  "core.uninstall.botNotFound": "Cannot find bot using the manifest ID %s",
+  "core.uninstall.confirm.tdp": "App registration of manifest ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.m365App": "M365 App of Title ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.bot": "Bot framework registration of bot ID: %s will be uninstalled. Please confirm.",
+  "core.uninstall.confirm.cancel.tdp": "App registration uninstallation cancelled.",
+  "core.uninstall.confirm.cancel.m365App": "M365 App uninstallation cancelled.",
+  "core.uninstall.confirm.cancel.bot": "Bot framework registration uninstallation cancelled.",
+  "core.uninstall.success.tdp": "App registration of manifest ID: %s has been successfully uninstalled.",
+  "core.uninstall.success.m365App": "M365 App of Title ID: %s has been successfully uninstalled.",
+  "core.uninstall.success.delayWarning": "The uninstallation of the M365 app might have some delay.",
```

Comment:
Proposed: The uninstallation of the M365 app may be delayed.

Reply:
Updated.